tests/python/mkl/test_mkldnn.py

```diff
@@ -309,15 +319,18 @@ def check_batchnorm_training(stype):
             data = mx.symbol.Variable('data', stype=stype)
             in_location = [mx.nd.array(data_tmp).tostype(stype), mx.nd.array(gamma).tostype(stype),
                            mx.nd.array(beta).tostype(stype)]
-            mean_std = [mx.nd.array(rolling_mean).tostype(stype), mx.nd.array(rolling_std).tostype(stype)]
+            mean_std = [mx.nd.array(rolling_mean).tostype(
```

Comment:
```suggestion
                        mean_std = [mx.nd.array(rolling_mean).tostype(stype), 
                                    mx.nd.array(rolling_std).tostype(stype)]
```
or 
```suggestion
                        mean_std = 
                                   [mx.nd.array(rolling_mean).tostype(stype), 
                                    mx.nd.array(rolling_std).tostype(stype)]
```
?

Reply:
Please have a look at this ~ there are a few fixes